docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst

```diff
@@ -1,184 +1,69 @@
-
-    # import WFST-based non-deterministic TN module
-    from nemo_text_processing.text_normalization.normalize_with_audio import NormalizerWithAudio
-
-    # initialize normalizer
-    normalizer = NormalizerWithAudio(
-            lang="en",
-            input_case="cased",
-            overwrite_cache=False,
-            cache_dir="cache_dir",
-        )
-    # try normalizer on a few examples
-    print(normalizer.normalize("123", n_tagged=10, punct_post_process=True))
-    # >>> {'one hundred twenty three', 'one hundred and twenty three', 'one twenty three', 'one two three'}
-
-
-
-To run this script with a .json manifest file, the manifest file should contain the following fields:
-Parameters to run audio-based normalization (more details could be found in `nemo_text_processing/text_normalization/normalize_with_audio.py <https://github.com/NVIDIA/NeMo/blob/stable/nemo_text_processing/text_normalization/normalize_with_audio.py>`__)
-
-.. list-table:: Parameters to run audio-based normalization
-   :widths: 10 10
-   :header-rows: 1
-
-   * - **Parameter**
-     - **Description**
-   * - **audio_data**
-     - path to the audio file
-   * - **text**
-     - raw text
-   * - **pred_text**
-     - ASR model prediction
-   * - **n_tagged**
-     - Number of tagged options to return, -1 - return all possible tagged options
-
-
-See `examples/asr/transcribe_speech.py <https://github.com/NVIDIA/NeMo/blob/stable/examples/asr/transcribe_speech.py>`__ on how to add ASR predictions.
-
-When the manifest is ready, run:
-
-.. code-block:: python
+.. note::
 
-    python normalize_with_audio.py \
-           --audio_data PATH/TO/MANIFEST.JSON \
-           --language en
+    Walk through `NeMo/tutorials/text_processing/Text_Normalization.ipynb <https://github.com/NVIDIA/NeMo/blob/stable/tutorials/text_processing/Text_Normalization.ipynb>`__ in `Google's Colab <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/text_processing/Text_Normalization.ipynb>`_.
 
 
-To run with a single audio file, specify path to audio and text with:
+Language Support
+-------------------
+Languages are listed in decreasing order of grammar complexity.
 
-    .. code-block:: python
+Deterministic TN: 
 
-        python normalize_with_audio.py \
-               --audio_data PATH/TO/AUDIO.WAV \
-               --language en \
-               --text raw text OR PATH/TO/.TXT/FILE
-               --model QuartzNet15x5Base-En \
-               --verbose
+* English
+* Spanish
+* German
 
-To see possible normalization options for a text input without an audio file (could be used for debugging), run:
+Deterministic ITN: 
 
-    .. code-block:: python
+* English
+* Spanish
+* German
+* French
+* Russian
+* Vietnamese
 
-        python python normalize_with_audio.py --text "RAW TEXT" --cache_dir "<PATH_TO_CACHE_DIR_TO_STORE_GRAMMARS>"
+Non-deterministic TN:
 
-Specify `--cache_dir` to generate .far grammars once and re-used them for faster inference.
+* English
```

Comment:
this could also be arranged in a table:
TN - Audio-based TN -ITN columns - and each language is a row